client/src/components/InputField.js

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+const InputField = styled.input`
+  height: 25px;
+  margin: 0;
+  border: none;
+  background: transparent;
+  outline: none;
```

Comment:
this will lower the accessibility "grade" of this website, you should consider just styling it differently. I believe this declaration will kick in once the `input` is focused (i.e. `focus` selector).